pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -494,9 +506,9 @@ func (reconciler *ReconcileKieApp) createCustomObject(obj v1.OpenShiftObject, cr
 	//TODO temp
 	bytes, err := yaml.Marshal(obj)
 	if err != nil {
-		fmt.Printf("Failed to marshall object %v", err)
+		logrus.Errorf("Failed to marshall object %v", err)
 	} else {
-		fmt.Printf("Will create\n\n%v\n\n", string(bytes))
+		logrus.Debugf("Will create\n\n%v\n\n", string(bytes))
```

Comment:
Please remove these 5-6 lines, they were only there for troubleshooting and if everything deploys fine now, they've outlived their usefulness.

Reply:
done